app-modules/etno/src/Models/Document.php

```
@@ -146,6 +148,22 @@ public static function localityRelations(): array
+        return Attribute::get(fn () => CitationFormatter::format(
+            title: $this->title,
+            subtitle: $this->subtitle,
+            authors: $this->authors->pluck('display_name'),
+            originators: $this->originators->pluck('person.display_name'),
+            publicationDate: $this->publication_date_start,
+            institutionName: $this->institution?->name,
+            type: $this->type,
+            timePeriodStart: $this->time_period_start,
+            timePeriodEnd: $this->time_period_end,
+            doi: $this->doi
+        ));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Database\Eloquent\Casts\Attribute' in method 'howToCite'. (undefined)

(StaticAccess)